python/tester/upload.py

```diff
@@ -175,8 +175,8 @@ def get_last_group(storage_client, instrument, date):
     Returns
     -------
     group : `int`
-        The largest existing group for ``instrument``, or a newly generated
-        group if none exist.
+        The largest existing group for ``instrument`` plus a random number,
+        or a newly generated group if none exist.
```

Comment:
In practice as the number is incremented before passing to the functions ( https://github.com/lsst-dm/prompt_prototype/blob/main/python/tester/upload.py#L157-L160 ), the addition of a random number here is not really necessary.  Mind if I remove it from here, so this really "gets last group" (instead of the last group plus something)? I don't think any other code uses this `upload.py`, right? 

If we want to keep some randomness, I can add a random number in `main` (for passing into `upload_from_raws` and `upload_from_random`).

Reply:
https://github.com/lsst-dm/prompt_prototype/pull/35/commits/08131a880d48787ba88282f4f22baff6586addb5